src/FadeIn.js

```diff
@@ -7,25 +7,28 @@ export default class FadeIn extends Component {
 
   componentDidMount() {
     const count = React.Children.count(this.props.children);
+    var customLag = parseInt(this.props.lag) ? parseInt(this.props.lag) : 50;
     let i = 0;
     this.interval = setInterval(() => {
       i++;
       if (i > count)
         clearInterval(this.interval);
 
       this.setState({ maxIsVisible: i });
-    }, 50);
+    }, customLag);
```

Comment:
Or maybe call the prop `delayStart`, so you have `delay` and `delayStart`. My guess is that `delay` is the more useful prop, but I'm not sure.